src/components/List/List.module.css

```diff
@@ -1,10 +1,21 @@
-.list{
-  --component-list-border_top_color: #BCC7CC;
+.list {
+  padding-left: 0;
+  list-style-type: none;
+}
```

Comment:
Since we do not set any default borderstyle, we now have a list variant with no border style. Is this something we need, or should we default the style to `solid`?

Reply:
I think default to solid yeah as we've esablished that we don't really want unstyled components